install.bat

```diff
@@ -1,4 +1,4 @@
-mkdir C:\securityTrainingScript
+mkdir -p C:\securityTrainingScript
```

Comment:
Can you point me too something showing this works on windows.

-p seems to be a linux only flag from what I can find on google, and my own testing.

The rest of the code in the repo is windows specific, so any fix has to work on windows.

Reply:
Maybe try something along the lines of:

https://stackoverflow.com/questions/4165387/create-folder-with-batch-but-only-if-it-doesnt-already-exist